src/main/java/com/spotinst/sdkjava/model/SpotinstAdminOrganizationClient.java

```diff
@@ -0,0 +1,331 @@
+            BaseSpotinstService.addCustomUserAgents(userAgentConfigurations);
+        }
+    }
+    //endregion
+
+    public User createUser(User userRequest ) {
+        return createUser(userRequest, false);
+    }
+
+    public User createUser(User userRequest, Boolean generateToken) {
+
+        User retVal = null;
+
+        RepoGenericResponse<User> creationResponse =
+                getSpotinstAccountAdminRepo().createUser(userRequest, generateToken, authToken);
+
+        if (creationResponse.isRequestSucceed()) {
+            retVal = creationResponse.getValue();
+        }
+        else {
+            List<HttpError> httpExceptions = creationResponse.getHttpExceptions();
+            HttpError       httpException  = httpExceptions.get(0);
+            LOGGER.error(
+                    String.format("Error encountered while attempting to create user. Code: %s. Message: %s.",
+                            httpException.getCode(), httpException.getMessage()));
+            throw new SpotinstHttpException(httpException.getMessage());
+        }
+        return retVal;
+    }
+
+    public List<OrganizationUsers> getOrganizationUsers() {
+
+        List<OrganizationUsers> retVal = null;
+
+        RepoGenericResponse<List<OrganizationUsers>> getResponse =
+                getSpotinstAccountAdminRepo().getOrganizationUsers(authToken);
+
+        if (getResponse.isRequestSucceed()) {
+            retVal = getResponse.getValue();
+        }
+        else {
+            List<HttpError> httpExceptions = getResponse.getHttpExceptions();
+            HttpError       httpException  = httpExceptions.get(0);
+            LOGGER.error(
+                    String.format("Error encountered while attempting to get organization users. Code: %s. Message: %s.",
+                            httpException.getCode(), httpException.getMessage()));
+            throw new SpotinstHttpException(httpException.getMessage());
+        }
+        return retVal;
+    }
+
+    public UserDetails getUserDetails(String userId) {
+
+        UserDetails retVal = null;
+
+        RepoGenericResponse<UserDetails> getResponse =
+                getSpotinstAccountAdminRepo().getUserDetails(userId, authToken);
+
+        if (getResponse.isRequestSucceed()) {
+            retVal = getResponse.getValue();
+        }
+        else {
+            List<HttpError> httpExceptions = getResponse.getHttpExceptions();
+            HttpError       httpException  = httpExceptions.get(0);
+            LOGGER.error(
+                    String.format("Error encountered while attempting to get user details for %s. Code: %s. Message: %s.",userId,
+                            httpException.getCode(), httpException.getMessage()));
+            throw new SpotinstHttpException(httpException.getMessage());
+        }
+        return retVal;
+    }
+
+    public Boolean updateUserGroupMapping(String groupId, List<String> userIds) {
```

Comment:
- Rename to updateUserGroupMappingOfUser
- Parameters are not correct - it should be userId and list of groupIds (it is not correct in documentation also).